src/EditorFeatures/Core.Wpf/Suggestions/FixAll/FixAllCodeFixGetFixesService.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    [ExportWorkspaceServiceFactory(typeof(IFixAllCodeFixGetFixesService), ServiceLayer.Host), Shared]
+    internal class FixAllCodeFixGetFixesService : IFixAllCodeFixGetFixesService, IWorkspaceServiceFactory
```

Comment:
This is an existing type that was renamed from `FixAllGetFixesService ` to `FixAllCodeFixGetFixesService` and some code split into a helper type `FixAllGetFixesServiceHelper` which is shared between `FixAllCodeFixGetFixesService` and  `FixAllCodeRefactoringGetFixesService`.

Reply:
why is it a workspaceservicefactory, instead of just an IWorkspaceService?

Reply:
This is existing code, and I have mimic'ed the same for `FixAllCodeRefactoringGetFixesService`. I can clean this up.